network/wsNetwork.go

```diff
@@ -1180,6 +1217,7 @@ func (wn *WebsocketNetwork) connectedForIP(host string) (totalConnections int) {
 }
 
 const meshThreadInterval = time.Minute
+const cliqueResolveInterval = 5 * time.Minute
```

Comment:
I'm not strongly opinionated regarding "how to express this constant". Only that "it needs to be a substantial amount of time without making any agreement progress". 
Using the SmallLambda implies that it would change with protocol upgrades, which seems to me like over engineering. I wouldn't mind thought moving this one into the config file, so that we could tweak this value over there.